nesspoll/package.json

```diff
@@ -5,7 +5,9 @@
   "dependencies": {
     "@fortawesome/fontawesome": "^1.1.5",
     "@fortawesome/react-fontawesome": "0.0.18",
+    "@types/material-ui": "^0.20.0",
     "@types/react-fontawesome": "^1.6.2",
+    "@types/react-tap-event-plugin": "0.0.30",
```

Comment:
All of the @types/ dependencies are development only dependencies. They are just making development experience better for us. The final build product is not dependent on them.

Reply:
so should I remove them from here?